specifications/catalog/example/dcat.distribution.example.json

```diff
@@ -0,0 +1,84 @@
+{
+    "@context":  "https://w3id.org/idsa/v5/context.json",
+    "@id": "http://provider.com/catalog1/dataset1/",
+    "@type": "dcat:Dataset",
+    "dct:title" : "IDS Dataset #1",
+    "dct:description": [ {
+        "@language": "en",
+        "@value": "This is the number 1 data asset of the IDS. The consumer must be a TRUST_PLUS certfified connector."
+    }, {
+        "@language": "de",
+        "@value": "Dies ist das Nummer 1 Data Asset im IDS. Der Consumer muss ein TRUST_PLUS zertifizierter Connetor sein."
+    } ],
+    "dcat:keyword": [
+        "IDS",
+        "data asset",
+        "dcat dataset"
+    ],
+    "dcat:distribution": [ {
+        "@type": "dcat:Distribution",
+        "dct:title" : "Distribution#1",
+        "dct:description": "This is a distribution which provides the data via a HTTP_REST dataplane.",
+        "dcat:keyword": [
+            "dataplane",
+            "public",
+            "distribution"
+        ],
+        "ids:dataPlaneKind": {
+            "@id": "idsc:HTTP_REST"
+        },
+        "dcat:accessService": [ {
+            "@type": [ "dcat:DataService", "idsc:NEGOTIATION_SERVICE" ],
+            "dcat:endpointURL": {
+                "@type": "xsd:anyURI",
+                "@value": "https://provider.com/connector/negotiation"
+            }
+        }, {
+            "@type": [ "dcat:DataService", "idsc:TRANSFER_SERVICE" ],
+            "dcat:endpointURL": {
+                "@type": "xsd:anyURI",
+                "@value": "https://provider.com/connector/transfer"
+            }
+        } ]
+    }, {
+        "@type": "dcat:Distribution",
+        "dct:title" : "Distribution#2",
+        "dct:description": "This is another distribution using a S3 dataplane.",
+        "dcat:keyword": [
+            "dataplane",
+            "public",
+            "distribution"
+        ],
+        "ids:dataPlaneKind": {
+            "@id": "idsc:S3"
+        },
+        "dcat:accessService": [ {
+            "@type": [ "dcat:DataService", "idsc:NEGOTIATION_SERVICE" ],
+            "dcat:endpointURL": {
+                "@type": "xsd:anyURI",
+                "@value": "https://provider.com/connector/negotiation"
+            }
+        }, {
+            "@type": [ "dcat:DataService", "idsc:TRANSFER_SERVICE" ],
+            "dcat:endpointURL": {
+                "@type": "xsd:anyURI",
+                "@value": "https://provider.com/connector/transfer"
+            }
+        } ]
+    } ],
+    "dcat:hasPolicy": {
+        "@type": "odrl:Offer",
+        "@id": "https://provider.com/edc/offer/1",
+        "odrl:permission": [ {
+            "odrl:action": {"@id": "odrl:use"},
+            "odrl:constraint" : [ {
+                "odrl:leftOperand": "idsc:SECURITY_PROFILE",
```

Comment:
This is more a discussion point. Since we are not defining specific policies for IDS yet, should we use a different namespace and example?